sdk/java/src/main/java/com/chain/api/CoreConfig.java

```diff
@@ -11,67 +11,135 @@
   public static class Info {
     public String state;
 
+    /**
+     * Whether the core has been configured.
+     */
     @SerializedName("is_configured")
     public boolean isConfigured;
 
+    /**
+     * RFC3339 timestamp reflecting when the core was configured.
```

Comment:
This is deserialized to a `Date`, so it's not RFC3339 in Java. Ditto for other instances of "RFC3339" in the comments.